src/Controller/Rest/ApiController.php

```diff
@@ -20,20 +22,20 @@ class ApiController extends FOSRestController
     */
     public function getCalendar()
     {
+        $sessionRepo = $this->getDoctrine()->getRepository(Session::class);
 
-        //Reikia paimti laisvus laikus is DB ir transforminti i apacioje esanti masyva
-        $dummyData = [
-            'Mon' => [
-                '0100-0140' =>  4,
-                '0500-0540' =>  2
-            ],
-            'Fri' => [
-                '0300-0400' =>  10,
-                '0500-0700' =>  16
-            ],
-        ];
-
-        return View::create($dummyData, Response::HTTP_OK);
+        $calendarInitColl = $sessionRepo->init_calendar();
```

Comment:
prasau, kaip galint greiciau pakeiskit situos dalykus kaip `init_calendar`, tai tikrai siaubingas naming'as, kuris ne tik, kad neatitinka stilistikos standartu, bet dar ir nepasako ka jis is esmes daro